packages/cli/src/handlers/dbt/run.ts

```diff
@@ -3,17 +3,13 @@ import execa from 'execa';
 import { LightdashAnalytics } from '../../analytics/analytics';
 import GlobalState from '../../globalState';
 import { generateHandler } from '../generate';
+import { DbtCompileOptions } from './compile';
 
-type DbtRunHandlerOptions = {
-    profilesDir: string;
-    projectDir: string;
-    target: string | undefined;
-    profile: string | undefined;
-    select: string[] | undefined;
-    models: string[] | undefined;
+type DbtRunHandlerOptions = DbtCompileOptions & {
```

Comment:
Some type aliases like `GenerateHandlerOptions` take advantage of `CompileHandlerOptions`. But, I feel `DbtCompileOptions` would look more natural in this case, as `DbtRunHandlerOptions` isn't exposed as command line options.